RecoTracker/ConversionSeedGenerators/plugins/ConversionSeedFilter.cc

```diff
@@ -17,31 +17,31 @@
 //
 
 #include <memory>
-#include "FWCore/Framework/interface/Frameworkfwd.h"
-#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "DataFormats/Math/interface/deltaPhi.h"
+#include "DataFormats/TrackReco/interface/Track.h"
+#include "DataFormats/TrackReco/interface/TrackFwd.h"
+#include "DataFormats/TrajectorySeed/interface/TrajectorySeedCollection.h"
+#include "FWCore/Framework/interface/ESHandle.h"
```

Comment:
Is this still needed?

Reply:
removed in [0a9ee2d](https://github.com/cms-sw/cmssw/pull/36980/commits/0a9ee2def74a0cf443e5e22a5b52be489d5a03de)